src/utility/Utility.cpp

```diff
@@ -91,3 +97,46 @@ std::vector<std::string> Utility::split(const char *text, char delimiter) {
     }
     return tokens;
 }
+
+#ifndef BENCHMARKING
+// ==================== Split string at character ==============================
+// *****************************************************************************
+/// \brief  creates a new named logger this function is only available if
+//          BENCHMARKING is not enabled
+/// \param  loggerName name of logger, represented in log file
+//          make sure you dont call this function twice with the same loggerName
```

Comment:
still up-to-date?

Reply:
nop thx for pointing it out. fixed in (5f84a7f)